src/components/shared/SearchInput.js

```diff
@@ -209,11 +275,20 @@ class SearchInput extends Component<Props, State> {
   renderNav() {
     const { count, handleNext, handlePrev } = this.props;
     if ((!handleNext && !handlePrev) || (!count || count == 1)) {
-      return;
+      return (
+        <div className="search-nav-buttons">
+          <span style={{ padding: "5px" }}>Modifiers:</span>
```

Comment:
We should use CSS for the padding to avoid duplication

Reply:
Okay